price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseProperties.java

```diff
@@ -33,8 +33,8 @@ public void setCostWeight(double costWeight) {
     public double costByPriceWeightInKg() {
         Properties price = new Properties();
         try {
-            ClassLoader classLoader = CostByPriceKgUseFile.class.getClassLoader();
-            InputStream is = classLoader.getResourceAsStream("price_kg.properties");
+            ClassLoader classLoader = CostByPriceKgUseProperties.class.getClassLoader();
+            InputStream is = classLoader.getResourceAsStream("price/price_kg.properties");
```

Comment:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Hi. I'm corrected. Now I often use it.